File: src/Circit.Weather.Api/Models/AstronomyModel.cs
namespace Circit.Weather.Api.Models
{
    public class AstronomyModel
    {
        public Location Location { get; set; }
        public Astronomy Astronomy { get; set; }
    }

    public class Astronomy
    {
        public Astro Astro { get; set; }
    }

    public class Astro
    {
        public string Sunrise { get; set; }
        public string Sunset { get; set; }
        public string Moonrise { get; set; }
        public string Moonset { get; set; }
        public string Moon_phase { get; set; }
        public int Moon_illumination { get; set; }
        public int Is_moon_up { get; set; }
        public int Is_sun_up { get; set; }
    }
}

File: src/Circit.Weather.Web/src/app/models/astronomy-model.ts
export interface AstronomyModel {
    location: {
        name: string;
        region: string;
        country: string;
        lat: number;
        lon: number;
        tz_id: string;
        localtime_epoch: number;
        localtime: string;
    },
    astronomy: {
        astro: {
            sunrise: string;
            sunset: string;
            moonrise: string;
            moonset: string;
            moon_phase: string;
            moon_illumination: number;
            is_moon_up: number;
            is_sun_up: number;
        }
    }
}
